backend/routes/itemRoutes.js

```diff
@@ -1,13 +1,15 @@
 import express from 'express';
-import { getItems, getItemById, createItem, updateItem, deleteItem } from '../controllers/itemController.js';
+import { getItems, getItemById, getMyItems, createItem, updateItem, deleteItem } from '../controllers/itemController.js';
 import { authMiddleware } from '../middlewares/authMiddleware.js';
 
 const router = express.Router();
 
+// Protect routes with authMiddleware
 router.get('/items', getItems);
 router.get('/items/:id', getItemById);
-router.post('/items', authMiddleware, createItem); // Protect this route with authMiddleware
-router.put('/items/:id', authMiddleware, updateItem); // Protect update if only logged-in users can update
-router.delete('/items/:id', authMiddleware, deleteItem); // Protect delete if only logged-in users can delete
+router.get('/my-items', authMiddleware, getMyItems);
+router.post('/items', authMiddleware, createItem);
+router.put('/items/:id', authMiddleware, updateItem);
+router.delete('/items/:id', authMiddleware, deleteItem);
 
 export default router;
```

Comment:
This optimizes model usage by preventing redundant recompilation in userModel.js and improves route security in itemRoutes.js. A new getMyItems route allows authenticated users to fetch their own items, and authMiddleware is now consistently applied to protect create, update, and delete operations.